### DISCUSSION TITLE
[更新游戏]看火人 Firewatch

### DISCUSSION THREAD

**Discussion by nukemiko** [Gamuxorg/bbs#295] [投递和请求游戏]
游戏名称：看火人
游戏名称(英文)：Firewatch
下载地址 :（如果有的话）：https://cloud.189.cn/t/FRvaEzf6V73y (访问码:jd1o)

[站内版本](https://www.linuxgame.cn/firewatch)：1.09
此处版本：1.12
游戏文件来源：GOG

**Comment by shouhuanxiaoji** [comment]
收到，正在处理

**Comment by shouhuanxiaoji** [comment]
已处理，感谢
